### ISSUE TITLE
[homeassistant.components.mobile_app.notify] Requested entity was not found. This message is generated externally to Home Assistant.

### ISSUE THREAD

**Issue by Mariusthvdb** [home-assistant/mobile-apps-fcm-push#38]
Is there any way we can debug this. In my current HA 2021.2.1 this is happening after each reload of template entities. Thought it might have to do with a closed iPhone device, but holding the open-end iOS app in front of me, our the latest MacOS app, this still is occurring.

Open for suggestion how to prevent/solve this.

<img width="636" alt="Schermafbeelding 2021-02-05 om 14 04 41" src="https://user-images.githubusercontent.com/33354141/107037271-21e2f780-67bb-11eb-8a3e-b1c6299e2c65.png">

link in the issues goes to Core, but this would be for the mobile_app?

**Comment by TomBrien** [comment]
This probably means the push token needs updating. Either that or there is a duplicate device issue in mobile app.

**Comment by Mariusthvdb** [comment]
so far, there are no duplicates , and I have restores Push-id's both on iOS app and MacOS app.

I did find an incorrect service mobile_app_wrongname, which was configured in a notification group, but that was corrected and the error persists.

Doesnt happen on HA restart, and only on template reloads.

At that same time a notification on my Mac pops up:

<img width="367" alt="Schermafbeelding 2021-02-05 om 15 49 47" src="https://user-images.githubusercontent.com/33354141/107049073-f4ea1100-67c9-11eb-88d0-afccd349bd50.png">

any thoughts you might have how to continue squashing this would be appreciated....
